river/Cursor.zig

```diff
@@ -105,6 +105,7 @@ pressed_count: u32 = 0,
 hide_cursor_timer: *wl.EventSource,
 
 hidden: bool = false,
+needWarp: bool = false,
```

Comment:
Struct fields should be snake case. I think `keyboard_focus_changed` might also be more helpful to future readers.

Reply:
I think it might be more versatile to name it this way in case there's some other case that needs mouse warpping